jobs/kola-upgrade.Jenkinsfile

```
@@ -72,7 +72,8 @@ currentBuild.description = "[${params.STREAM}][${params.ARCH}] - ${start_version
 def cosa_memory_request_mb = 1024
 if (params.ARCH == 'x86_64') {
     // local (qemu+x86_64) testing will require more memory
-    cosa_memory_request_mb = 3584
+    // bios=1024, uefi=1024, uefi-secure=1024, overhead=512
```

```suggestion
    // bios=1024, uefi=1024, uefi-secure=1536, overhead=512
```
Just to make the math work. Did we have a 512 overhead before, or does uefi-secure require a little bit more memory?

---

basically before we didn't allow for any overhead. Each VM could use up to their provided memory, which when added up was all the pod itself was given. the kola processes and qemu processes and helpers use a little memory, not sure how much exactly, but it's some, so we'd end up using too much and something from our cgroup would need to get killed.

So we probably don't actually need 512, but it's a nice round number.

---

Thanks for the clarification. The suggested change ensures the comment matches the assignment statement. The comment adds up to the original `3584`, but `cosa_memory_request_mb` is assigned `4096`.

---

ahh - I missed the comment was out of alignment

---

fixed